### ISSUE TITLE
add comments to explain where libraries come from in the arduino code?

### ISSUE THREAD

**Issue by rwhiffen** [bjoerngiesler/BB-R2_STEM_ESP32#2]
Opening this as an issue rather than a pull request for now.

It might be useful to add comments in the arduino code to explain where the libraries come from.
// sketch makes use of https://github.com/STEMpedia/DabbleESP32 library
#include <DabbleESP32.h>

//sketch makes use of https://github.com/DFRobot/DFRobotDFPlayerMini
#include <DFRobotDFPlayerMini.h>

Something like that?

**Comment by Maciej-droid** [comment]
Yes this information should be good in code or in README "Install Dabble32, DFRobotDFPlayerMini and ESP32 ESP32S2 AnalogWrite libraries in ArduinoIDE".
